build.psm1

```diff
@@ -1087,7 +1112,7 @@ function Start-PSPester {
     }
 
     $PSFlags = @("-noprofile")
-    if (-not [string]::IsNullOrEmpty($ExperimentalFeatureName)) {
+    if (-not (Test-IsPreview (& $powershell -version)) -and -not [string]::IsNullOrEmpty($ExperimentalFeatureName)) {
```

Comment:
We cannot do this. In `test\tools\TestMetadata.json`, you will find:
```
{
  "ExperimentalFeatures": {
    "Microsoft.PowerShell.Utility.PSDebugRunspaceWithBreakpoints": ["test/powershell/Modules/Microsoft.PowerShell.Utility/New-PSBreakpoint.Tests.ps1"],
    "ExpTest.FeatureOne": [ "test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1" ]
  }
}
```
The experiment feature name `ExpTest.FeatureOne` is for testing the experimental feature code. By making changes here, then the `Experimental Feature Basic Tests - Feature-Enabled` tests in `ExperimentalFeature.Basic.Tests.ps1` will never run.

Side note: the experimental feature `Microsoft.PowerShell.Utility.PSDebugRunspaceWithBreakpoints` was never defined in the utility module, so this feature doesn't really exist ... ;)

Reply:
I pushed a commit to address this.

Reply:
@SteveL-MSFT All CIs passed. Let me know if you have any concerns about my changes.

Reply:
Looks fine. Thanks.